main/main.cpp

```diff
@@ -605,6 +605,9 @@ void Main::print_help(const char *p_binary) {
 	print_help_option("--gpu-abort", "Abort on graphics API usage errors (usually validation layer errors). May help see the problem if your system freezes.\n", CLI_OPTION_AVAILABILITY_TEMPLATE_DEBUG);
 #endif
 	print_help_option("--generate-spirv-debug-info", "Generate SPIR-V debug information. This allows source-level shader debugging with RenderDoc.\n");
+#if defined(DEBUG_ENABLED) || defined(DEV_ENABLED)
+	print_help_option("--extra-gpu-memory-tracking", "Enables additional memory tracking. See the manual about RenderingDevice::get_driver_and_device_memory_report et. al. This functionality may not be available for all APIs.\n");
```

Comment:
```suggestion
	print_help_option("--extra-gpu-memory-tracking", "Enables additional memory tracking. See the class reference about `RenderingDevice.get_driver_and_device_memory_report()` and other linked methods. Currently only implemented for Vulkan.\n");
```

Reply:
I've changed it to your version, but I had to add more text (a warning that it can cause crashes and this is a known issue we cannot fix).